server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -50,6 +55,7 @@
 import io.crate.testing.UseJdbc;
 
 @IntegTestCase.ClusterScope(numDataNodes = 2)
+@UseRandomizedOptimizerRules(value = 1, disablePercentage = 1, alwaysKeep = RewriteInsertFromSubQueryToInsertFromValues.class)
```

Comment:
`RewriteInsertFromSubQueryToInsertFromValues` is needed otherwise a lot of the Tests in `InsertIntoIntegrationTest` fail.